libvuln/libvuln.go

```diff
@@ -162,3 +169,17 @@ func (l *Libvuln) GCFull(ctx context.Context) (int64, error) {
 
 	return i, err
 }
+
+// Initialized reports whether the backing vulnerability store is initialized.
+func (l *Libvuln) Initialized(ctx context.Context) (bool, error) {
+	ok := l.initialized.Load().(bool)
```

Comment:
instead of having l.initialized, can store.Initialized just return an atomic bool?